image_tools/src/cam2image.cpp

```diff
@@ -193,7 +194,7 @@ int main(int argc, char * argv[])
         cv::waitKey(1);
       }
       // Publish the image message and increment the frame_id.
-      std::cout << "Publishing image #" << i << std::endl;
+      std::cout << "Publishing image #" << i << " to topic " << topic << std::endl;
```

Comment:
If you could instead output the topic once on startup, the tests for these executables shouldn't have to be updated https://github.com/ros2/demos/blob/ec3affc96896f338e125be8ba553de92851956f2/image_tools/test/cam2image.regex#L1
(the tests are disabled but will be re-enabled in https://github.com/ros2/demos/pull/83)